### PR TITLE
added node_id indexes

### PR DESCRIPTION
**Pull request by johndickinson**
dsc-243

### CODE REVIEW

File: tools/hedgehog_database_create.sh.in [hunk 1]
```diff
@@ -257,6 +257,7 @@ do
         ADD CONSTRAINT \"pk_data_${SERVER}_${PLOT}_${DNOW}\" PRIMARY KEY (server_id, node_id , plot_id , starttime , key1, key2 ),
         ADD CONSTRAINT \"fk_data_${SERVER}_${PLOT}_${DNOW}_node\" FOREIGN KEY (node_id) REFERENCES dsc.node (id) MATCH SIMPLE ON UPDATE NO ACTION ON DELETE NO ACTION,
         ADD CONSTRAINT \"fk_data_${SERVER}_${PLOT}_${DNOW}_plot\" FOREIGN KEY (plot_id) REFERENCES dsc.dataset (id) MATCH SIMPLE ON UPDATE NO ACTION ON DELETE NO ACTION;" | sudo -u $hhconfig_database_owner psql -d $hhconfig_database_name 
+      echo "CREATE INDEX \"idx_data_${SERVER}_${PLOT}_${DNOW}_node_id\" ON dsc.\"data_${SERVER}_${PLOT}_${DNOW}\" (node_id);" | sudo -u $hhconfig_database_owner psql -d $hhconfig_database_name
```
Comment (saradickinson) [inline tools/hedgehog_database_create.sh.in]: Did you mean to leave the echo there?

File: tools/hedgehog_database_manage_partitions.sh.in [hunk 2]
```diff
@@ -108,6 +108,7 @@ ALTER TABLE dsc.data_${server}_${plot}_${NOW}
         ADD CONSTRAINT pk_data_${server}_${plot}_${NOW} PRIMARY KEY (server_id, node_id , plot_id , starttime , key1, key2 ),
         ADD CONSTRAINT fk_data_${server}_${plot}_${NOW}_node FOREIGN KEY (node_id) REFERENCES dsc.node (id) MATCH SIMPLE ON UPDATE NO ACTION ON DELETE NO ACTION,
         ADD CONSTRAINT fk_data_${server}_${plot}_${NOW}_plot FOREIGN KEY (plot_id) REFERENCES dsc.dataset (id) MATCH SIMPLE ON UPDATE NO ACTION ON DELETE NO ACTION;
+CREATE INDEX idx_data_${server}_${plot}_${now}_node_id ON dsc.data_${server}_${plot}_${now} (node_id);
```
Comment (saradickinson) [inline tools/hedgehog_database_manage_partitions.sh.in]: (minor) indentation is wrong

File: tools/hedgehog_database_create.sh.in (hunk 1 above)
Comment (johndickinson) [inline tools/hedgehog_database_create.sh.in]: yes see line 256

File: tools/hedgehog_database_manage_partitions.sh.in (hunk 2 above)
Comment (johndickinson) [inline tools/hedgehog_database_manage_partitions.sh.in]: I don't think so

**Pull request merged**